test/fluentd-forward.sh

```diff
@@ -75,6 +75,10 @@ update_current_fluentd() {
     port 24284\n\
    </server>\n\
   </store>\n\
+  <filter **>\n\
+    @type record_modifier\n\
+    remove_keys _id\n\
```

Comment:
should this be _openshift_es_genid ?

Reply:
No I dont believe so.  The gen plugin creates the id and stuffs it in a key that you define.  It then uses that to populate the _id field which is what ES uses.  We then tell it to remove the original key because apparently ES doesnt like that.  In our case are not getting a duplicate record so we need to remove the '_id' field

Reply:
Also, use a `record_transformer` filter instead of `record_modifier`